components/Chips/src/TypographyThemer/MDCChipViewTypographyThemer.m

```diff
@@ -14,11 +14,23 @@
 
 #import "MDCChipViewTypographyThemer.h"
 
+#import "MaterialApplication.h"
+#import "MaterialTypography.h"
+
 @implementation MDCChipViewTypographyThemer
 
 + (void)applyTypographyScheme:(nonnull id<MDCTypographyScheming>)typographyScheme
                    toChipView:(nonnull MDCChipView *)chipView {
   chipView.titleFont = typographyScheme.body2;
+  if (typographyScheme.mdc_adjustsFontForContentSizeCategory) {
+    UIContentSizeCategory sizeCategory = UIContentSizeCategoryLarge;
+    if (@available(iOS 10.0, *)) {
+      sizeCategory = chipView.traitCollection.preferredContentSizeCategory;
+    } else if ([UIApplication mdc_safeSharedApplication]) {
+      sizeCategory = [UIApplication mdc_safeSharedApplication].preferredContentSizeCategory;
+    }
+    chipView.titleFont = [chipView.titleFont mdc_scaledFontForSizeCategory:sizeCategory];
```

Comment:
Rather than setting the property twice, consider storing an variable initially populated with `typographyScheme.body2` that is then modified if `typographyScheme.mdc_adjustsFontForContentSizeCategory` is enabled. The resulting variable can then be assigned to titleFont.

Reply:
Addressed in 41e42f329072582edc9a8b49d524c2f90a19ebe5.